documentation/book/cluster-operator.adoc

```diff
@@ -673,6 +673,115 @@ INFO:: The Cluster Operator doesn't validate the provided configuration. When in
 Kafka Connect cluster might not start or might become unstable. In such cases, the configuration in the `connect-config` field
 should be fixed and the Cluster Operator will roll out the new configuration to all Kafka Connect instances.
 
+===== Kafka Connect S2I deployment
+
+When using {ProductName} together with an {OpenShiftName} cluster, Kafka Connect can be deployed with support for https://docs.openshift.org/3.9/dev_guide/builds/index.html[{OpenShiftName} Builds] and https://docs.openshift.org/3.9/creating_images/s2i.html#creating-images-s2i[Source-to-Image (S2I)].
```

Comment:
Is Kafka Connect the subject of the dependent clause, "When using {ProductName} together with an {OpenShiftName} cluster"?

If not, I think we should rephrase to:  "When using {ProductName} together with an {OpenShiftName} cluster, a user can deploy Kafka Connect with support for https://docs.openshift.org/3.9/dev_guide/builds/index.html[{OpenShiftName} Builds] and https://docs.openshift.org/3.9/creating_images/s2i.html#creating-images-s2i[Source-to-Image (S2I)].